test/dynamo/test_triton_kernels.py

```diff
@@ -924,6 +924,40 @@ def f(x, y):
         compiled_out = torch.compile(f)(x, y)
         self.assertEqual(compiled_out, eager_out)
 
+    @requires_cuda
+    @skipIfRocm
+    @torch._dynamo.config.patch(capture_dynamic_output_shape_ops=True)
+    @torch._dynamo.config.patch(capture_scalar_outputs=True)
+    @common_utils.parametrize("backend", ["eager", "aot_eager", "inductor"])
+    def test_triton_kernel_dynamic_shape_tensor(self, backend):
```

Comment:
I would actually suggest `unbacked` rather than `dynamic` in the name here, that's what we've been using to identify tests that involve data dependent; it disambiguates between u0 and s0.